tests/network/expose.go

```diff
@@ -734,36 +680,22 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			table.DescribeTable("[label:masquerade_binding_connectivity]Should verify the exposed service is functional before and after VM restart.", func(ipFamily ipFamily) {
 				skipIfNotSupportedCluster(ipFamily)
 				vmExposeArgs = appendIpFamilyToExposeArgs(ipFamily, vmExposeArgs)
-
-				vmObj := vm
-
-				By("Exposing a service to the VM using virtctl")
-				virtctl := tests.NewRepeatableVirtctlCommand(vmExposeArgs...)
-				Expect(virtctl()).ToNot(HaveOccurred(), "should succeed exposing a service via `virtctl expose ...`")
+				executeVirtctlExposeCommand(vmExposeArgs)
 
 				vmi := startVMWithServer(virtClient, "tcp", testPort)
 				Expect(vmi).NotTo(BeNil(), "should have been able to start the VM")
 
-				By("Getting back the service's allocated cluster IP.")
-				svc, err := virtClient.CoreV1().Services(vmObj.Namespace).Get(context.Background(), serviceName, k8smetav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				Expect(validateClusterIp(svc.Spec.ClusterIP, ipFamily)).To(Succeed())
-
-				By("Iterating over the ClusterIPs")
-				serviceIPs := svc.Spec.ClusterIPs
-				for ipOrderNum, ip := range serviceIPs {
-					assert.XFail(xfailError, func() {
-						Expect(createAndWaitForJobToSucceed(runHelloWorldJob, vmObj.Namespace, ip, servicePort, fmt.Sprintf("%d ClusterIP", ipOrderNum+1))).To(Succeed())
-					}, ipOrderNum > 0)
-				}
+				vmObj := vm
```

Comment:
IIUC, `vm` is created by `createStoppedVM`, which returns `*v1.VirtualMachine` - a pointer. What's the point of using coyping the pointer instead of using `vm` directly?

Reply:
I see no reason to keep it, but it can be addressed in a different PR as it seems to be out of scope of this one